test/npm.js

```diff
@@ -108,13 +108,11 @@ describe("createDependencyVersion", () => {
 		expect(result).to.eql(semver)
 	})
 
-	it("throws otherwise", async () => {
+	it("throws otherwise", () => {
 		let nonSemver = "$0.$0.0-0.0.0-0.0.0-yeet$17"
-		let version = `blablahblah#${nonSemver}`
-		expect(npm.createDependencyVersion(["yeet", version])).to.eventually.throw()
-		expect(
-			npm.createDependencyVersion(["yeet", nonSemver])
-		).to.eventually.throw()
+		return npm.createDependencyVersion(["yeet", nonSemver]).catch(error => {
```

Comment:
What would happen if this resolved? Would it fail the test?

Reply:
oof no it wouldn't.

Reply:
the new one would !

Reply:
wait, i haven't pushed it yet

Reply:
...

Reply:
the new one would !